### ISSUE TITLE
Mobs that Player disguised with  remains after player die

### ISSUE THREAD

**Issue by Kardane** [samolego/MobDisguises#25]
To reproduce:
1. disguise with any mob
2. kill yourself
3. and mob remains with unkillable state
Though you can select that mob, but fail to execute command to it

![2022-07-30_13 25 42](https://user-images.githubusercontent.com/64183602/181872337-d418c3d4-d06f-437d-b60b-4f2b686986ac.png)

**Comment by samolego** [comment]
Sounds like a visual bug, thanks for reporting.
Can others see it as well? (I assume not ..?)

**Comment by Kardane** [comment]
No only disguised player can see it. It would be client bug.
Btw it is possible to make mob invisible to owner? Maybe it isnt possible?

**Comment by samolego** [comment]
You mean to hide the "self disguise"?

**Comment by Kardane** [comment]
Yes, When disguised as creeper, everyone can see creeper including disguised players themselves. I thought it would be nice to hide mob from disguised player view only, but it would be impossible to make invisible to certain players...

**Comment by samolego** [comment]
Ah, yeah, you mean `truesight` option. I've been working on that, but it's in buggy state atm.

**Comment by Archer-Dante** [comment]
I see same bug =(
1.18.2 fabric
